src/sdl/surface.cpp

```diff
@@ -31,14 +33,23 @@ void add_refcount(surface& surf)
 void free_surface(surface& surf)
 {
 	if(surf) {
-		SDL_FreeSurface(surf);
+		SDL_DestroySurface(surf);
 	}
 }
 
 void make_neutral(surface& surf)
 {
-	if(surf && surf->format->format != SDL_PIXELFORMAT_ARGB8888) {
-		surf = surf.clone();
+	if(surf) {
```

Comment:
Merge artifact. Should be able to just revert this block.

Reply:
`surf->format->format` doesn't exist anymore.

Reply:
Looking into this further, it looks the check should just be
```cpp
If(surf && surf->format == SDL_PIXELFORMAT_ARGB8888)
```
According to the migration docs:
> SDL_PixelFormatEnum has been renamed SDL_PixelFormat and is used instead of Uint32 for API functions that refer to pixel format by enumerated value.

That is, SDL_Surfaces no longer store a pointer to a format struct, just a enum value. So we don’t need to do any extra checking with the format details.